### DISCUSSION TITLE
Is it possible to test dynamic select box options using `->live()`?

### DISCUSSION THREAD

**Discussion by bzy107** [filamentphp/filament#9535] [Help] [answered]
I want to test the options displayed for "sub_category" when selecting any item under the "category" as shown on this URL: 

https://filamentphp.com/docs/3.x/forms/advanced#dependant-select-options

However, I'm not sure how to write the test.
thank you.

**Comment by leandrocfe** [comment, marked as answer]
You could try something like this

```php
use function Pest\Livewire\livewire;

it('can validate dependent select', function () {

    livewire(YourCreatePage::class)
        ->fillForm([
            'category' => 'web',
        ])
        ->assertSee('frontend_web')
        ->assertSee('backend_web');
        
});
```

**Reply by bzy107** [reply]
Thank you. It's been resolved.
